pandas/tests/groupby/test_groupby.py

```diff
@@ -2146,3 +2146,13 @@ def test_groupby_duplicate_columns():
     result = df.groupby([0, 0, 0, 0]).min()
     expected = DataFrame([["e", "a", 1]], columns=["A", "B", "B"])
     tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_series_with_tuple_name():
+    # GH 37755
+    s = Series([1, 2, 3, 4], index=[1, 1, 2, 2], name=("a", "a"))
+    s.index.name = ("b", "b")
+    result = s.groupby(level=0).last()
```

Comment:
is anything about this specific to `last`?

Reply:
No, in fact the regression would raise in the call to `groupby`, I just took last from the issue that was raised. Should something else be done here?

Reply:
> Should something else be done here?

no, just curious